### DISCUSSION TITLE
【求助】问个问题

### DISCUSSION THREAD

**Discussion by lbghhj** [share121/inter-knot#500] [其他]
13.8和13.11到底哪个大呀

**Comment by swyh51982** [comment]
13.11大。四位数比三位数大，这不是常识吗？

**Reply by guodawd** [reply]
因为13.8＝13.80
又因为80》11
所以13.8》13.11
由此得证（确信）

**Reply by swyh51982** [reply]
唉，没想到我纵横数学界几百秒，竟还会犯这种错误。

**Comment by lllliufeng** [comment]
其实13.8<13.11但是13.80>13.11
